### PR TITLE
New Homework; Final

### PR DESCRIPTION
**Pull request by frania1**
In order to get the changes into README.md on GitHub I pressed save and then commit then push the information to GitHub. Before I committed, I used R Studio's "preview" option to determine whether my code was formatted correctly. I ran into some challenges in the commitment stage because I kept forgetting to push the README.md to GitHub.  From this activity, I realized the importance of line spacing between code and between words, and after a few tries, I managed to finally get the code to work. Lastly, I can see why people choose to write in Markdown, there is no need to click anything, it seems like once you know the codes, you can just type away everything.

### CODE REVIEW

**Review by Yanchi-Liu — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |          X |              |            |
| **Coding strategy**         |          X |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |          X |              |            |
| **Ease of access**          |   X        |              |            |

Very good job!
You have accomplished all the tasks required with a high quality. 
Hope every assignment of your future well!

**Review by Mirrorlkj — COMMENTED** [review]
Evaluation
----------------------------------------------------
  | Topic                       | Excellent | Satisfactory | Needs Work |
  |-----------------------------|-----------|--------------|------------|
  | **Coding style**            |     X      |              |            |
  | **Coding strategy**         |     X      |              |            |
  | **Presentation**            |      X     |              |            |
  | **Achievement, creativity** |      X     |              |            |
  | **Ease of access**          |      X     |              |            |

Remarks:

* Nice work completing all the requirements!
* Photo and links are appropriately integrated into the text
* Just for future reference, you may include your name in the title

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello! My name is Frania Mendoza Lua, I am a second year **PHD student** at the *School of Social Administration*. 
+
+I study:
+
+1. immigration
```
Comment (bensoltoff) [inline README.md:7]: You don't have to number each list element. Markdown will do that automatically.

```
1. immigration
1. teens
1. teen health behaviors
```

1. immigration
1. teens
1. teen health behaviors

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+# Why I study teens?
+
+I study teens because it is such an important developmental period that shapes the trajectory of our lives. For example, check out this [embarassing video](https://www.youtube.com/watch?v=OBZ_jYmedBI) of me doing my first interview on TV as a teen, it haunts me til this day.  
+
+# The Weather
+
+## OMG Chicago weather
+
+As a Southern Californian, growing up 15 minutes (30 with traffic) from the beach, living in Chicago is brutal. Naively, I chose a place 3 blocks from the lake thinking, I made it, I finally live by the water. It's a sham! I didn't know what a lake effect was until now. Living by the lake is nice for like two months, the rest of it is a mixture of sadness, coldness, and the ever ending temptation of wanting to take a walk on the lakefront but realizing its March. I low key get offended when Chicagoans call the lake a beach, and don't you dare call it a coast.
```
Comment (bensoltoff) [inline README.md:25]: My brother-in-law moved to LA (from Northern Virginia) a few years ago. It did not take him long to acclimate. Now if it's in the 60s he has to wear a scarf and jacket to stay warm

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |         x     |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |       x    |              |            |

**Pull request closed without merging**